packages/jsii-calc/lib/submodule/my-class.ts

```diff
@@ -9,4 +10,8 @@ export class MyClass implements nested_submodule.deeplyNested.INamespaced {
   public allTypes?: AllTypes;
 
   public constructor(public readonly props: SomeStruct) {}
+
+  public methodWithSpecialParam(param: SpecialParameter) {
```

Comment:
Can we add a method that returns a type from a separate module as well? I'm not clear if the current solution is handling the dependencies of the method return types or not?

Reply:
the return types should already be handled but yeah I can add a separate test in jsii-calc.